File: Iniciante/1065 - Pares entre Cinco Números/TypeScript/1065.ts
import * as fs from "fs";

const NUMBER_OF_VALUES: number = 5;

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.trim().split("\n");
  let numberOfEvenValues: number = 0;
  for (let i = 0; i < NUMBER_OF_VALUES; i++) {
    const value: number = parseInt(lines[i], 10);
    if (value % 2 === 0) {
      numberOfEvenValues++;
    }
  }
  console.log(`${numberOfEvenValues} valores pares`);
}

main();

File: Iniciante/1065 - Pares entre Cinco Números/Python/1065.py
NUMBER_OF_VALUES = 5


def main():
    number_of_even_values = 0
    for _ in range(NUMBER_OF_VALUES):
        value = int(input())
        if value % 2 == 0:
            number_of_even_values += 1
    print(f"{number_of_even_values} valores pares")


if __name__ == "__main__":
    main()
